common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -109,7 +109,8 @@ private static async Task Run(Type testType, PerfOptions options)
 
                         if (options.Warmup > 0)
                         {
-                            await RunTestsAsync(tests, options.Sync, options.Parallel, options.Rate, options.Warmup, "Warmup");
+                            await RunTestsAsync(tests, options.Sync, options.Parallel, options.Rate, options.Warmup, options.StatusInterval,
```

Comment:
This gets very tedious to maintain as new options are added, especially if in multiple places like this seems to be. Just pass `options` and let the callee (in this case, 1 it looks like) get what they need.

Reply:
#16970